components/IscnRegisterForm.vue

```diff
@@ -850,6 +866,17 @@ export default class IscnRegisterForm extends Vue {
     return 'CreativeWork'
   }
 
+  get defaultLanguage() {
+    const containsChinese = (str: any) => /[\u4E00-\u9FFF\u3400-\u4DBF\uF900-\uFAFF]/.test(str);
+
+    const isEpubInChinese = this.epubMetadata?.language === 'zh'
+
+    const titleContainsChinese = this.name ? containsChinese(this.name) : isEpubInChinese;
+    const descriptionContainsChinese = this.description ? containsChinese(this.description) : isEpubInChinese;
+
+    return (titleContainsChinese || descriptionContainsChinese) ? 'zh' : 'en';
+  }
```

Comment:
Cleaner to just at the end `return (isEpubInChinese  || titleContainsChinese || descriptionContainsChinese) ? 'zh' : 'en'` and don't use `isEpubInChinese` in 874 and 875

Reply:
[🎨 Simplify default language check](https://github.com/likecoin/app-like-co/pull/471/commits/7013a40c386ad473a0d62cd0b2a72b6d17512961)